File: Beecrowd/1. Beginner/1065/1065.cpp
#include <iostream>
using namespace std;
int main () {
    
    int n;
    int count = 0;
    
    for(int i=0; i < 5; i++) {
        cin >> n;
        if(n % 2 == 0) {
            count++;
        }
    }
    cout << count << " valores pares" << endl;
    
return 0;
}

File: Beecrowd/1. Beginner/1065/1065.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner input = new Scanner(System.in);

        double number;
        int count = 0;

        for(int i=0; i<5; i++) {
            number = input.nextDouble();
            if(number % 2 == 0) {
                count++;
            }
        }

        System.out.println(count + " valores pares");

        input.close();
    }
}
